src/app/groups/group-set-manager/group-set-manager.component.html

```diff
@@ -0,0 +1,59 @@
+<!-- <group-selector
```

Comment:
Why is this commented out? You should still be using group-selector. But that component would need to be migrated to Angular 17 first